recipes/aioprocessing/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "aioprocessing" %}
+{% set author = "dano" %}
+{% set version = "2.0.1" %}
+{% set sha256 = "1fe91c0dca9d98e1a1280c120a7c29841f2b79fe9789239adb0d93d5200a0b60" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ author }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install --use-pep517  . -vvv .
+  number: 0
+
+requirements:
+  host:
+    - python >=3.5
+    - flit-core
+    - pip
+  run:
+    - python >=3.5
+
+test:
+  imports:
+    - aioprocessing
+  source_files:
+    - runtests.py
+    - tests
+  commands:
+    - pip check
+    - python -m coverage run --branch runtests.py -v --failfast
+    - conda install multiprocess  # [not osx]
```

Comment:
If you require this package, please add it to `requires` below. As this is a pure Python package, it's fine to only run an import test if it is difficult otherwise.

Reply:
Just being a bit too zealous. I removed the multiprocess test